src/libs/state.js

```diff
@@ -1031,6 +1032,7 @@ const state = new Vue({
                     realname: user.realname || '',
                     modes: user.modes || '',
                     away: user.away || '',
+                    colour: TextFormatting.createNickColour(user.nick),
```

Comment:
This going to kill performance as you could get a nicklist of thousands of nicks and colour generation is expensive. This should be generated only when needed